ui/component/or-chart/src/index.ts

```diff
@@ -370,6 +371,9 @@ export class OrChart extends translate(i18next)(LitElement) {
     @property({type: Object})
     public assetAttributes: [number, Attribute<any>][] = [];
 
+    @property({type: Array}) // The indexes of the assetAttributes array that appear on the right Y axis.
+    public readonly rightAxisAttributes: AttributeRef[] = [];
+
```

Comment:
Maybe more appropriate to use a dedicated data type for an attribute on a chart which could contain all config options:

```
{
  "attributeRef": ...,
  "secondaryAxis": true,
 ...
}
```

Can see the benefit of having a separate property to avoid breaking existing usages.

Why is this property readonly?

Reply:
Yeah indeed had a dilemma on how to approach this.
Might be benefitial for future proofing, so I will update this to a separate type as you described.

Made it readonly since I thought it was odd to change this value within `or-chart` itself.
However, now thinking about it, I think it doesn't do any harm at all. I will update this as well.